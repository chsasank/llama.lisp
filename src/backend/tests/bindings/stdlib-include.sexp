(c-lisp
    ,@(include
        (stdio.h stdlib.h string.h) ; Headers
        (malloc puts strcpy); Functions to include
        () ; Structs to include
        ()) ; Typedefs

    (define ((main void))
        (declare buf (ptr int8))
        (set buf (call malloc (sext 50 int64)))
        (call strcpy buf "Brought to you by malloc, strcpy and puts")
        (call puts buf)))
