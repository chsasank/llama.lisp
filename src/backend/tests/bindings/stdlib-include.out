Brought to you by malloc, strcpy and puts
